backend/src/test/java/codezap/category/service/facade/MemberCategoryApplicationServiceTest.java

```diff
@@ -0,0 +1,180 @@
+                    .hasMessage("이름이 " + categoryName + "인 카테고리가 이미 존재합니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("멤버에 따른 카테고리 조회")
+    class FindAllByMember {
+        @Test
+        @DisplayName("멤버에 따른 카테고리 조회 성공")
+        void findAllByMemberSuccess() {
+            Member member = memberRepository.save(MemberFixture.memberFixture());
+            Member otherMember = memberRepository.save(MemberFixture.createFixture("gugu"));
+            Category category1 = categoryRepository.save(new Category("카테고리 1", member));
+            Category category2 = categoryRepository.save(new Category("카테고리 2", member));
+            Category category3 = categoryRepository.save(new Category("카테고리 3", otherMember));
+
+            FindAllCategoriesResponse response = sut.findAllByMember(member.getId());
+
+            assertThat(response.categories()).hasSize(2)
+                    .containsExactly(FindCategoryResponse.from(category1), FindCategoryResponse.from(category2))
+                    .doesNotContain(FindCategoryResponse.from(category3));
+        }
+
+        @Test
+        @DisplayName("멤버에 따른 카테고리 조회 실패 : 멤버 존재하지 않음")
+        void findAllByMemberFailNotExistsMember() {
+            Long notExistsId = 100L;
+
+            assertThatThrownBy(() -> sut.findAllByMember(notExistsId))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + notExistsId + "에 해당하는 멤버가 존재하지 않습니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("카테고리 수정")
+    class Update {
+        @Test
+        @DisplayName("카테고리 수정 성공")
+        void updateSuccess() {
+            Member member = memberRepository.save(MemberFixture.memberFixture());
+            MemberDto memberDto = MemberDto.from(member);
+            Category category = categoryRepository.save(new Category("카테고리 1", member));
+            String renameCategoryName = "카테고리 수정";
+            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);
+
+            sut.update(memberDto, category.getId(), updateCategoryRequest);
+
+            Category actual = categoryRepository.fetchById(category.getId());
+            assertAll(
+                    () -> assertThat(actual).isEqualTo(category),
+                    () -> assertThat(actual.getName()).isEqualTo(renameCategoryName)
+            );
+        }
+
+        @Test
+        @DisplayName("카테고리 수정 실패 : 멤버 존재하지 않음")
+        void updateFailNotExistsMember() {
+            MemberDto memberDto = MemberDto.from(MemberFixture.memberFixture());
+            String renameCategoryName = "카테고리 수정";
+            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);
+
+            assertThatThrownBy(() -> sut.update(memberDto, 1L, updateCategoryRequest))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + memberDto.id() + "에 해당하는 멤버가 존재하지 않습니다.");
+        }
```

Comment:
```suggestion
        @Test
        @DisplayName("카테고리 수정 실패 : 멤버 존재하지 않음")
        void updateFailNotExistsMember() {
            MemberDto nonExistentMemberDto = MemberDto.from(MemberFixture.memberFixture());
            String renameCategoryName = "카테고리 수정";
            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);

            assertThatThrownBy(() -> sut.update(nonExistentMemberDto, 1L, updateCategoryRequest))
                    .isInstanceOf(CodeZapException.class)
                    .hasMessage("식별자 " + memberDto.id() + "에 해당하는 멤버가 존재하지 않습니다.");
        }
```

이 테스트코드만 보고 무엇이 실패의 원인인지 파악하기 어려운 것 같아요. 그래서 다음과 같이 제안합니다~